### PR TITLE
DM-38830: make rules.Heartbeat and rules.Enabled alarm severity configurable

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -20,4 +21,3 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
-        args: [--settings-path=pyproject.toml]
```
Comment (wvreeven) [inline .pre-commit-config.yaml:23]: Since ts_pre_commit_conf is now used, this file may be deleted.

Comment (wvreeven) [reply .pre-commit-config.yaml:23]: Don't forget to update `pyproject.toml` as well and remove the configuration for `flake8` and `isort`.

File: python/lsst/ts/watcher/rules/enabled.py [hunk 2]
```diff
@@ -59,7 +59,29 @@ def __init__(self, config):
...
     @classmethod
     def get_schema(cls):
-        schema_yaml = """
+        def make_severity_property(state, default_severity):
+            """Make all the data for one {state}_severity property.
+
+            Parameters
+            ----------
+            state : `salobj.State`
+                The state
```
Comment (wvreeven) [inline python/lsst/ts/watcher/rules/enabled.py:68]: Please terminate in a `.`.

**Review by wvreeven — APPROVED** [review]
Looks good to me. I especially like how the code was cleaned up by introducing a method that creates a config.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (r-owen) [reply .pre-commit-config.yaml:23]: I will ask you to review this again to make sure I did it right.

**Review by wvreeven — APPROVED** [review]
Looks good to me.

**Pull request merged**